stable/redis/values.yaml

```diff
@@ -26,6 +26,21 @@ usePassword: true
 ##
 # redisPassword:
 
+# Define where redis should get its secrets from.
+# On default, it uses the generated secret by itself, but changing `secretKeyRefs` allows you
+# to get the data from your own secret resource. Won't be used if `usePassword == false`
+#
+# Syntax:
+# ```
+# secretKeyRefs:
+#   <container-env-var-name>:
+#     name: <secret resource name> (optional, defaults to redis' charts secret as resource name)
+#     key: <secret resource entry>
+# ```
+secretKeyRefs:
+  REDIS_PASSWORD:
```

Comment:
I have this concern: what if someone removes this env var? Could there be a conflict with that and usePassword?